pkg/kubelet/kubelet.go

```diff
@@ -1894,7 +1904,12 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Mount volumes and update the volume manager
-	podVolumes, err := kl.mountExternalVolumes(pod)
+	// Default limits for containers here to have downward API expose user-friendly limits to pods.
+	defaultedPod, err := kl.defaultPodLimitsForDownwardApi(pod)
+	if err != nil {
+		return err
+	}
+	podVolumes, err := kl.mountExternalVolumes(defaultedPod)
```

Comment:
Nooooooooo..... Can this PR go in before that?

On Fri, Jun 10, 2016 at 12:23 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/27107#discussion_r66667864>
> :
>
> > @@ -1894,7 +1904,12 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
> >  	}
> >
> >  	// Mount volumes and update the volume manager
> > -	podVolumes, err := kl.mountExternalVolumes(pod)
> > +	// Default limits for containers here to have downward API expose user-friendly limits to pods.
> > +	defaultedPod, err := kl.defaultPodLimitsForDownwardApi(pod)
> > +	if err != nil {
> > +		return err
> > +	}
> > +	podVolumes, err := kl.mountExternalVolumes(defaultedPod)
>
> Just a note -- this code is completely changing in another PR now --
> rebase alert.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27107/files/e546b39782552a2af2edac8d228d8c6b12052566#r66667864>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKLHLNeaGncEUQx2UnohZnwOKEHTNks5qKbm4gaJpZM4Ixkx3>
> .
>